### PR TITLE
[Quantum Chess] Improve test failure output

### PR DESCRIPTION
**Pull request by losos0**
See 2nd commit message for an example.

### CODE REVIEW

File: unitary/alpha/quantum_world.py [hunk 1]
```diff
@@ -111,9 +111,9 @@ def copy(self) -> "QuantumWorld":
         for remap in self.qubit_remapping_dict:
             new_dict = {}
             for key_obj, value_obj in remap.items():
-                new_dict[
-                    new_world.get_object_by_name(key_obj.name)
-                ] = new_world.get_object_by_name(value_obj.name)
+                new_dict[new_world.get_object_by_name(key_obj.name)] = (
+                    new_world.get_object_by_name(value_obj.name)
+                )
```
Comment (dstrain115) [inline unitary/alpha/quantum_world.py:116]: What's up with all these formatting changes?  Are you on the same version of black as everyone else?

**Review by dstrain115 — COMMENTED** [review]
Test error message change looks good, but I think we should revert the formatting changes (or maybe put them in a separate PR) since they are not related to this change.

File: unitary/alpha/quantum_world.py (hunk 1 above)
Comment (losos0) [reply unitary/alpha/quantum_world.py]: There is a CI check on this repo that fails if the formatting is not identical to that which whichever black version the workflow installs produces, so I had to add a formatting commit to make it pass.

In the mean time, Peter already hit the same issue and merged a commit making those formatting changes, so now I can remove mine.

Comment (dstrain115) [reply unitary/alpha/quantum_world.py]: ok, perfect.  I will merge this then.

**Pull request merged**